src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -127,8 +125,6 @@ object Boot extends IOApp with LazyLogging {
         appConfig.samDatabaseConfig.samBackground
       )
 
-      googleSynchronizerExecutionContext <- ExecutionContexts.fixedThreadPool[IO](24)
```

Comment:
since running things serially via `IO` this execution context is no longer needed